Resources/Prototypes/Entities/Objects/Weapons/Guns/Launchers/launchers.yml

```diff
@@ -144,3 +144,43 @@
       tags:
         - CartridgeRocket
     proto: ImmovableRodSlow
+
+- type: entity
+  name: pirate cannon
+  parent: BaseWeaponLauncher
+  id: WeaponLauncherPirateCannon
+  description: Kaboom!
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Launchers/pirate_cannon.rsi
+    layers:
+      - state: icon
+        map: ["enum.GunVisualLayers.Base"]
+  - containers:
+      balistic-ammo: !type:Container
+        ents: []
+      ballistic-ammo: !type:Container
+        ents: []
+    type: ContainerContainer
+  - type: Clothing
+    sprite: Objects/Weapons/Guns/Launchers/pirate_cannon.rsi
+  - type: Gun
+    fireRate: 1
+    selectedMode: FullAuto
+    availableModes:
+      - FullAuto
```

Comment:
SemiAuto?

Reply:
only has 1 shot so it doesnt make a huge difference if its auto or semiauto

Reply:
this was fixed